plugins/CoreHome/Controller.php

```diff
@@ -277,8 +277,21 @@ public function redirectToPaypal()
                 unset($parameters[$name]);
             }
         }
+        $paypalParameters = [
+            "cmd" => "_s-xclick"
+        ];
+        if (empty($parameters["onetime"]) || $parameters["onetime"] != "true") {
+            $paypalParameters["hosted_button_id"] = "DVKLY73RS7JTE";
+            $paypalParameters["currency_code"] = "USD";
+            $paypalParameters["on0"] = "Piwik Supporter";
```

Comment:
do we need to change something in paypal setting to make it work for Matomo Supporter?

Reply:
Probably. Using `Matomo Supporter` just shows a Paypal error. I guess this is the name of the paypal "product".

Reply:
@mattab could you add this product to PayPal or configure it somehow?